dist/elemica-suggest.js

```diff
@@ -6,18 +6,22 @@ elemicaSuggest 0.9.2-SNAPSHOT
  */
 
 (function() {
+  var indexOf = [].indexOf || function(item) { for (var i = 0, l = this.length; i < l; i++) { if (i in this && this[i] === item) return i; } return -1; };
```

Comment:
Why do we need to define `indexOf` here? Starting from IE9 it should be available in `Array`: https://www.w3schools.com/jsref/jsref_indexof_array.asp

Is it for IE8 compatibility? I think we can drop a support for this version in the `elemica-suggest`.

Reply:
No idea, it's what `npm run-script dist` produced. I didn't change any option related to IEs

Reply:
Interesting... Let's trust coffee's compiler, we have no better option :wink: